tox.ini

```
@@ -2,9 +2,9 @@
 min_version = 4.22
 
 env_list =
-    python{3.9,3.10,3.11}-dj42-wagtail{63}
-    python{3.10,3.11,3.12}-dj51-wagtail{63,70}
-    python{3.13}-dj52-wagtail70
+    py{3.9,3.10,3.11}-dj42-wagtail{63,70,71}
+    py{3.10,3.11,3.12}-dj51-wagtail{63,70,71}
+    py3.13-dj52-wagtail{70,71}
```

suggestion: now that the matrix combinations are becoming large again, I suggest simplifying things a bit

```suggestion
    py{3.12}-dj51-wagtail{63,70,71}
    py3.13-dj52-wagtail{70,71}
```